src/tabs/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classnames from 'classnames';
+
+const Tabs = ({ active, children }) => (
+  <nav className="govuk-tabs">
+    <ul>
+      {
+        React.Children.map(children, (child, index) => (
+          <li className={classnames({ active: index === active })}>{ child }</li>
```

Comment:
Will React beef about these not having `key`s?

Reply:
no, it seems to be cleverer when using `React.Children.map`

Reply:
Clever girl.